types/jquery_countdown/tslint.json

```diff
@@ -0,0 +1,18 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "max-line-length": false,
```

Comment:
These other lint rules should be followed too.

Reply:
Things like max line length are a coding preference and according to the [contribution guide](http://definitelytyped.org/guides/contributing.html#code-style) there is no coding style requirements for definitely typed. Since I will be maintaining this code, I want this code to match the style I use for code base. Do you have a good reason to remove any of these? If not, please merge this pull request.

Reply:
The contribution guide was written before we added linting and hasn't been updated since then. For over a year now we've been linting all new packages.

Reply:
It seems like the real issue is with the documentation, not my code. If you can update the guide I will be happy to make the change. If not, please merge this pull request as it follows all the official guide lines.

Reply:
@andy-ms @weswigham Are you planning to update the guide so I can follow the new guide lines or are you planning to accept this pull request which follows the current guide lines?